INSTALL.md

```diff
@@ -669,22 +672,58 @@ ClamAV has two bytecode runtimes:
    We ran out of time in 0.104 development to update to support newer versions
    of LLVM. LLVM 3.6.2 is the newest version supported in ClamAV 0.104.
 
+#### Interpreter Bytecode Runtime
+
 At the moment, the *interpreter* is the default runtime, while we work out
 compatibility issues with newer versions of libLLVM. This default equates to:
 
 ```sh
 cmake .. -D BYTECODE_RUNTIME="interpreter"
 ```
 
-If you wish to build using LLVM instead of the intereter, you will need to
-obtain v3.6 of the LLVM development libraries. Then build using these options:
+#### LLVM JIT Bytecode Runtime
+
+If you wish to build using LLVM JIT for the bytecode runtime instead of the
+bytecode interpreter, you will need to install the LLVM development libraries.
+ClamAV currently supports LLVM versions 8.0 through 12.0.
 
+To build with LLVM for the bytecode runtime, use these options, specifying the
+paths and versions appropriate for your LLVM installation:
 ```sh
 cmake .. \
   -D BYTECODE_RUNTIME="llvm"       \
-  -D LLVM_ROOT_DIR="/opt/llvm/3.6" \
-  -D LLVM_FIND_VERSION="3.6.0"
+  -D LLVM_ROOT_DIR="/opt/llvm/8.0" \
```

Comment:
I never use LLVM_ROOT_DIR or LLVM_FIND_VERSION in building.  If you only have 1 llvm installed, it may not be necessary.  Do we want to move them down a little and say something like "If llvm is not detected, try these"?

Reply:
Good call!